qiita_db/data.py

```diff
@@ -416,6 +416,17 @@ def _remove_filepath(self, fp, conn_handler, queue):
             self._set_link_filepaths_status("failed: %s" % msg)
             raise QiitaDBError(msg)
 
+        # The filepath belongs to one or more studies
+        studies_linked = self.studies
```

Comment:
I don't think that behavior is desired. A user could link 3 files (because he forgot to add one of the barcodes e.g.) and he just needs to add one more. The point that I'm making is that if you're not able to unlink files because the raw data is used in another study, you should not be able to link them either. Does that make sense?